docs/roadmap/2022-08-back-for-more.md

```diff
@@ -86,6 +86,7 @@ If you've been following our GitHub discussions, or the Analytics Engineering ro
 
 1. **Multi-project deployments.** The ability to `ref` a model from someone else's project, without having had to run it first. Split up monolithic projects of 5000 models into 10 projects of 500, grouped by team and domain. This is more than just "namespacing": to really solve for this, we also need to solve for versioning and contracts, and support a variety of deployment mechanisms. The discussion for this has been in [#5244](https://github.com/dbt-labs/dbt-core/discussions/5244); I'll have more to share over the next few months.
 2. **External orchestration.** The same dbt DAG, playing a more active role. We've been developing this idea internally, and have arrived at a few strong opinions. This would not a new node type, but an upgrade to the ones we already have: sources, models, and exposures. Sources that can trigger their own ingest. Exposures that can trigger downstream consumers (syncs, sinks, etc). Models that can define and run transformations in companion execution environments, that read from and write back to the centralized data warehouse. For each external integration, a simple request where it possible, and a dedicated plugin where justified. If you're someone who followed along the original "external nodes" discussion ([#5073](https://github.com/dbt-labs/dbt-core/discussions/5073))—especially if you've got a tool you'd be excited to integrate into dbt's DAG—let's talk.
+3. **Next steps for Python.** Supporting data science workflows, machine learning model trianing, and "MLOps"? Adding more adapaters and Python runtimes? Enbabling writing docs and tests directly in `.py` files? There's a lot to consider here, and we need your help! We've started [#5752](https://github.com/dbt-labs/dbt-core/discussions/5742) to kick off the discussion.
```

Comment:
I moved this further down, under "existing capabilities" - but it's an arbitrary distinction. Happy to have it here instead. Let's just pick one or the other